src/utils/settings.ts

```diff
@@ -3,8 +3,11 @@ import {getValueFromLS} from './utils';
 export const userSettings = window.userSettings || {};
 export const systemSettings = window.systemSettings || {};
 
+export const settingsApi = window.web_version ? systemSettings.settingsApi : undefined;
```

Comment:
What if we define settingsApi ourselves if it isn't defined initially? We know methods it should provide and can define it. Thus in other code there will be no need to worry, just use settingsApi.

Reply:
Could you please clarify you question? I didn't get it

If you mean changing `settingsApi` on a go, this case isn't supported. The problem is that when we declare new `settingsApi` by ourselves, all currently stored settings should be copied there. Otherwise, there will be scenario, where our settings partly stored in `localStorage`, partly in other store, it will be a mess.

If you mean incapsulating this logic into some instance, that will provide certain API and decide where to store data within itself, I don't think it's really needed. The problem, that I solve here, is that we want to use `localStorage` both in internal and in embedded version. Now it will be a migration period, after which `settingsApi` will be deleted

Currently, it's first iteration to allow UI usage in Nebius, where there is no special backend service for user settings. After that I add some migration to code, that will copy data from `settingsApi` to `localStorage`. And as final step, use only `localStorage`

Reply:
Ok, I've got it! Lets stick to current version!